src/zoo.js

```diff
@@ -11,57 +11,187 @@ eslint no-unused-vars: [
 
 const data = require('./data');
 
-function animalsByIds(ids) {
-  // seu código aqui
-}
+const { animals, employees, prices, hours } = data;
 
-function animalsOlderThan(animal, age) {
-  // seu código aqui
-}
+const animalsByIds = (...ids) => animals.filter(animal => ids.some(id => animal.id === id));
 
-function employeeByName(employeeName) {
-  // seu código aqui
-}
+const animalsOlderThan = (name, age) => {
+  const older = animals.find(animal => name === animal.name).residents.every(res => res.age >= age);
+  return older;
+};
+
+const employeeByName = (eName) => {
+  const employeeFiltered = employees.find(e => e.firstName === eName || e.lastName === eName);
+  return {
+    ...employeeFiltered,
+  };
+};
+
+const createEmployee = ({ id, firstName, lastName }, { managers, responsibleFor }) => ({
```

Comment:
Curti o jeito que você usou o abbreviation object literal na função! Daria pra fazer usando o spread operator também : 
```suggestion
const createEmployee = (personalInfo, associatedWith) => ({ ...personalInfo, ...associatedWith });
```